package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "vscode-ros",
-    "version": "0.4.5-dev",
+    "version": "0.4.5",
```

Comment:
Should this to be "0.4.5-dev"?

Reply:
our current release version is `0.4.5` so the next version should be `0.4.6-dev`. I'm thinking about updating the version number to `0.4.6-dev` in your change, so the version number only changes when there is an actual change (versioning still matters since our `master` is continuously being built and released on azure devops)